src/modules/data/blocks/rsvp/sagas.js

```diff
@@ -456,8 +456,12 @@ export function* updateRSVPHeaderImage( action ) {
 		 */
 		yield put( actions.setRSVPIsSettingsLoading( true ) );
 		yield put( ticketActions.setTicketsIsSettingsLoading( true ) );
+
+		const post_type = wpSelect('core/editor').getCurrentPostType();
+		const rest_base = ( 'tribe_events' === post_type ) ? 'tribe_events' : 'posts';
```

Comment:
One concern here is if the user puts this block on any other post type other than events or posts (pages or other custom post types), this will break. Can we make the `rest_base` depend on the post type completely?

Reply:
Hm, good point!